server/api/crud/model_monitoring/model_endpoints.py

```diff
@@ -496,7 +496,11 @@ def verify_project_has_no_model_endpoints(self, project_name: str):
         if not mlrun.mlconf.igz_version or not mlrun.mlconf.v3io_api:
             return
 
-        endpoints = self.list_model_endpoints(project_name)
+        try:
+            endpoints = self.list_model_endpoints(project_name)
+        except mlrun.errors.MLRunInvalidArgumentError:
+            # store not found, there is nothing to delete
```

Comment:
How do you know that's the underlying error?

Reply:
can be done inside